client/src/components/Ticketing/mainpage/hero.tsx

```diff
@@ -15,10 +15,29 @@ const Hero = () => {
   });
   const dispatch = useAppDispatch();
 
+  const ref = useRef(null);
+
   useEffect(()=>{
     dispatch(fetchTicketingData());
   }, []);
 
+  const handlePrevious = () => {
+    if (ref.current) {
+      ref.current.scrollBy({
+        left: -640,
+        top: -450,
```

Comment:
Is there any particular reason why top is scrolled?

Reply:
Originally meant for desktop vertical navigation, but since we hide the buttons on `sm` it would be considered unnecessary.